docs/pages/getting_started.rst

```diff
@@ -1,55 +1,309 @@
+   - ``praw_username`` 
+   - ``praw_password``
+
+   For example, you can invoke your scirpt like this:
+
+.. code-block:: shell
+
+    praw_username=bboe praw_password=not_my_password python my_script.py
+
+2) in the ``praw.ini`` file you provide. The section name for these settings
+   should be specified with an environment variable named ``praw_site``; if no
+   such environment variable is set, the default section name is ``DEFAULT``.
+   You can put your ``praw.ini`` file in one or both of the following places 
+   (both will be read if present):
+   
+   1. the working directory when you invoke your script
+   2. your OS's config directory (for Linux, this is ``$XDG_CONFIG_HOME`` or 
+      ``$HOME/.config``; for Windows, this is ``${APPDATA}``)
+
+   If you don't know how to write ini files, follow `this example`_.
+
+.. _`this example`: https://github.com/praw-dev/praw/blob/praw4/praw/praw.ini
+
+Get a subreddit
+---------------
+
+To get a ``Subreddit`` instance, all you need to know is the subreddit's
+display name. Pass that name when calling the ``subreddit`` method of your
+``Reddit`` instance. For example:
+
+.. code-block:: python
+
+    # assuming you have a Reddit instance referenced by reddit
+    subreddit = reddit.subreddit("redditdev")
 
-Iterating Over Submissions
---------------------------
+    print(subreddit.display_name) # Output: redditdev
+    print(subreddit.title) # Output: reddit Development
+    print(subreddit.description) # Output: A subreddit for discussion of ...
 
-There are two primary methods for iterating through submissions.
+Get submissions from a subreddit
+--------------------------------
 
-The first method is to pick a subreddit, and then iterate through one of its
-many generators of submissions, e.g., ``hot``, ``new``, or ``top``. In the
-following example we build a list of submissions which may be used for later
-processing.
+Now that you have a ``Subreddit`` instance, you can get some submissions
+(``Submission`` instances) from it! There are several ways of sorting all the
+submissions of a subreddit: hot, new, top, etc. A ``Subreddit`` instance has a
+method for each of these sorting approaches, namely, these:
+
+    ``controversial``
+    ``gilded``
+    ``hot``
+    ``new``
+    ``rising``
+    ``top``
+
+Each of these methods will return ``ListingGenerator``, something that you can
+iterate through. For example:
 
 .. code-block:: python
 
-   submissions = []
-   for submission in reddit.subreddit('python').hot():
-       submissions.append(submission)
+    # assuming you have a Subreddit instance referenced by subreddit
+    for submission in subreddit.hot(limit=10):
+        print(submission.title) # Output: the title of the submission
+        print(submission.ups) # Output: upvote count
+        print(submission.downs) # Output: downvote count
+        print(submission.id) # Output: the ID of the submission
+        print(submission.url) # Output: the URL of the submission
```

Comment:
This should be the URL that the submission points to, which if it is a self post will be the submission's URL.